packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -210,6 +210,26 @@ function Title( { item, categoryId } ) {
 	}
 	return (
 		<HStack alignment="center" justify="flex-start" spacing={ 2 }>
+			<Flex
+				as="div"
```

Comment:
Not directly related to this PR, but I think this is unnecessary since the `Flex` component is rendered as a `div` element by default.

Reply:
Yup, let's fix it separately.